generator/integration_tests/golden/v1/golden_kitchen_sink_client.h

```diff
@@ -21,6 +21,7 @@
 
 #include "generator/integration_tests/golden/v1/golden_kitchen_sink_connection.h"
 #include "google/cloud/future.h"
+#include "google/cloud/internal/make_status.h"
```

Comment:
Seems like we get this `#include` whether it is used or not.

Reply:
Removed if not used.